parachains/runtimes/assets/statemint/src/xcm_config.rs

```diff
@@ -317,7 +319,12 @@ impl xcm_executor::Config for XcmConfig {
 	type MaxAssetsIntoHolding = MaxAssetsIntoHolding;
 	type AssetLocker = ();
 	type AssetExchanger = ();
-	type FeeManager = ();
+	type FeeManager = XcmFeesToAccount<
+		Self,
+		polkadot_common::RelayOrOtherSystemParachains<Runtime>,
+		AccountId,
+		TreasuryAccount,
```

Comment:
What about sending it to the `PotId`? That's the account used by the collators in `pallet_collator_selection` to get the rewards. AFAIK treasury account is not used in Statemint/e.

Reply:
But it will be :)